### PR TITLE
chore: Add backup and restore steps

### PR DESCRIPTION
**Pull request by jnikhila**
chore-add-backup-restore-steps

## Checklist
I have:
- [ ] run the content through Grammarly
- [ ] linked to sample apps when relevant
- [ ] added the meta description for each page in the PR
- [ ] minimized the callouts and added only when necessary
- [ ] added the `queryString` parameter to the Tabs (if used)
- [ ] masked PII in images. For example, login credentials, account details, and more
- [ ] added images only when necessary
- [ ] deleted the images that are no longer used for the updated pages in the PR
- [ ] followed the image file naming convention while renaming or adding new images. (Use lowercase letters, dashes between words, and be as descriptive as possible)
- [ ] used the `<figure/>` tag instead of a markdown representation for images
- [ ] added the `<figcaption/>` tag to add a caption to the image
- [ ] added the `alt` attribute in the `` tag
- [ ] verified and updated the cross-references or created redirect rules for the changed or removed content 
- [ ] reviewed and applied the style changes for UI formatting. For example, Bold the UI elements(Buttons on screen) used in the doc.

### CODE REVIEW

File: website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ Follow the steps below to configure Appsmith to use an external MongoDB instance
 - A self-hosted Appsmith instance. See the [installation guides](/getting-started/setup/installation-guides) for installing Appsmith.
 - Ensure that your external MongoDB has a replica set configuration in place. Connect to your database as an admin user and run [rs.initiate()](https://docs.mongodb.com/manual/reference/method/rs.initiate/). Note that if you're using MongoDB Cloud, the replica set configuration is already set up for you.
 - Ensure the MongoDB user account has `readWrite` and `clusterMonitor` roles assigned.
+- At least 2 GB of free storage space for backup and update tasks.
+- Create a backup of the Appsmith instance, See [Backup instance](/getting-started/setup/instance-management/appsmithctl#backup-instance) guide. (Recommended)
```
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md:24]: @sharat87 - Added the step for backup.

Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md:24]: I presume we have to add steps for restoring the data from embedded  MongoDB to External. Could you please share those steps.

Comment (sharat87) [reply website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md:24]: We have a doc on both backup and restore, right? Can we instead mention something like this?

> Before you switch your MongoDB, please use the Backup and Restore guide to copy your data from your old MongoDB to the new one.

Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md:24]: Yes, we can. So they don't need to take the backup of instance - which we are doing for all the other docs. In this case, we will just link them to - [Backup database](https://docs.appsmith.com/getting-started/setup/instance-management/appsmithctl#backup-database) and[ Restore database](https://docs.appsmith.com/getting-started/setup/instance-management/appsmithctl#restore-database)

Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md]: Reworded. Review.

**Review by sharat87 — APPROVED** [review]

**Pull request merged**